packages/storage/src/indexed-db/index.ts

```diff
@@ -63,7 +64,9 @@ export class IndexedDb implements IndexedDbInterface {
 
         // TODO: To implement
```

Comment:
Is this todo still relevant? 🤔

Reply:
I agree, `NOTES` does not require a change, and `SWAPS` is done
Removed todo